src/utils/schemas/main/CryptoKitties/index.ts

```diff
@@ -22,18 +21,18 @@ export const CryptoKittiesSchema: Schema<CryptoKittiesType> = {
   assetFromFields: (fields: any) => fields.ID,
   assetToFields: (asset) => ({ ID: asset }),
   formatter: async (asset) => {
-    const response = await axios
-      .get(`https://api.cryptokitties.co/kitties/${asset}`)
-      .catch((err) => {
-        if (
-          err.response &&
-          (err.response.status === 404 || err.response.status === 400)
-        ) {
-          return null;
-        } else {
-          throw err;
-        }
-      });
+    const response = await fetch(
```

Comment:
do we even need this custom logic for cyrpto kitties?

Reply:
Unfortunately, I think we do because:
1. CryptoKitties are traded on OpenSea.
2. We have a `getBalance` function which calls the on-chain contracts
3. The `getBalance` function needs to get the schema
4. CryptoKitties was a contract released before modern erc721 formats